environment-wasm-host.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - https://repo.mamba.pm/emscripten-forge
```

Comment:
Ahhh we need to use the correct channel here. I think all updated packages built with the new version are being uploaded to 
`https://repo.prefix.dev/emscripten-forge-dev`

See here https://prefix.dev/channels/emscripten-forge-dev/packages/cppinterop